mobile/src/main/res/values-de/strings.xml

```diff
@@ -103,4 +103,9 @@
     <!-- drawer -->
     <string name="drawer_open">Sitemap-Bereich öffnen</string>
     <string name="drawer_close">Sitemap-Bereich schließen</string>
+    <string name="minus">-</string>
```

Comment:
Do you need to specify  "+" "-" "settings_openhab_icon_format_png"  "settings_openhab_icon_format_svg" here?  Or will this be inherited from mobile/src/main/res/values/strings.xml ? I'm asking because I'm not sure either.

Reply:
The problem is: I dont know every language supported by Android. If only one language is using a different sign for "plus" or different letter for PNG, we cannot make it non-translatable.
It is not needed to add it to the german translation, but it reduces lint errors

Reply:
understood, i'm fine with that